webpack/assets/javascripts/compute_resource/openstack.js

```diff
@@ -0,0 +1,34 @@
+import $ from 'jquery';
+import { showSpinner, hideSpinner } from '../foreman_tools';
+
+export function schedulerHintFilterSelected(item) {
+  let filter = $(item).val();
+
+  if (filter === '') {
+    $('#scheduler_hint_wrapper').empty();
+  } else {
+    let url = $(item).attr('data-url');
+    // eslint-disable-next-line no-undef
+    let data = serializeForm().replace('method=patch', 'method=post');
+
+    showSpinner();
+    $.ajax({
+      type: 'post',
+      url,
+      data,
+      complete() {
+        hideSpinner();
+      },
+      error(jqXHR, status, error) {
+        $('#scheduler_hint_wrapper').html(
+          // eslint-disable-next-line no-undef
+          Jed.sprintf(__('Error loading scheduler hint filters information: %s'), error)
```

Comment:
Where does this come from? Don't we need to include `Jed`?

Reply:
its currently included from the asset pipelne gettext gem. we would need to break it out in another pr.

Reply:
How does this work for jest tests, then?

Reply:
@timogoebel until now, we used mock or similar functions to not actually call the translation functions.

Reply:
@timogoebel see https://github.com/webhippie/gettext_i18n_rails_js/blob/master/lib/assets/javascripts/gettext/all.js for more details. IMHO this should not block this PR.